### PR TITLE
BEING ABLE TO RUN ASSIGNMENT RULES

### PR DESCRIPTION
**Pull request by deneuxa**
#### Why?
TO BE ABLE TO RUN ASSIGNMENT RULES

#### How?
Using the API

### CODE REVIEW

File: lib/intercom/service/conversation.rb [hunk 1]
```diff
@@ -52,6 +52,13 @@ def assign(reply_data)
         assignee_id = reply_data.fetch(:assignee_id) { fail 'assignee_id is required' }
         reply reply_data.merge(message_type: 'assignment', assignee_id: assignee_id, type: 'admin')
       end
+
+      def run_assignment_rules(id)
```
Comment (jonnyom) [inline lib/intercom/service/conversation.rb]: Looks good! It'd be cool to get a test for this behaviour as well? 😄

Comment (deneuxa) [reply lib/intercom/service/conversation.rb]: 😲We already pay you a lot for you product man.
Recently we had to debug your api that was sending wrong data on the feedback webhook.

We can code for you as well, but maybe you could thanks us with a free month then 😉

Comment (jonnyom) [reply lib/intercom/service/conversation.rb]: Unfortunately that's not up to me 😄 the SDKs are a community driven project.

>Recently we had to debug your api that was sending wrong data on the feedback webhook.

Have you reached out to support around this?

**Comment by deneuxa** [comment]
yes they corrected your API after we gave the demonstration of the wrong
behaviour ;)

**Comment by SeanHealy33** [comment]
Closed due to inactivity.

**Pull request closed without merging**